index.html

```diff
@@ -214,5 +214,52 @@ <h1>About Me</h1>
         </div>
       </section>
     </main>
+    <footer>
+      <section class="get-in-touch">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </section>
+      <form action="https://formspree.io/f/meqdgwge" method="POST">
+        <div class="form-group">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input type="email" name="email" placeholder="Email" required />
+          <textarea
+            cols="33"
+            rows="3"
+            name="message"
+            placeholder="Enter text here"
+            required
+          ></textarea>
+        </div>
+        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly add Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. It will make your design stick to the Figma design :+1: 

## Color to use for your button interaction 

![image](https://user-images.githubusercontent.com/95297251/192661645-48be74f2-aa34-41dc-be5f-fce8f6eda9f4.png)